source/unit/Fixture.ooc

```diff
@@ -170,9 +170,10 @@ Fixture: abstract class {
 }
 
 TestFailedException: class extends Exception {
+	description: String
```

Comment:
I think now there is some kind of a buffer pointer, which is null only for string literals, so it works kind of like a boolean flag. I may be wrong, though.